### PR TITLE
Clarify: Apollo also supports Mutation Batching

### PR DESCRIPTION
**Pull request by ignacio-chiazzo**
Clarify that Apollo also supports Mutation Batching. It wasn't clear in the documentation. 

https://github.com/apollographql/apollo-client/issues/1253

https://github.com/apollographql/react-apollo/issues/1222

### CODE REVIEW

File: source/network.md [hunk 1]
```diff
@@ -359,8 +359,10 @@ Apollo lets you automatically batch multiple queries into one request when they
 
 To use batching, simply pass a `BatchedNetworkInterface` to the `ApolloClient` constructor. You can do so by using  `createBatchingNetworkInterface` instead of `createNetworkInterface`. `createBatchingNetworkInterface` takes a single options object (the same as `createNetworkInterface`) with an additional `batchInterval` option, which determines how long (in milliseconds) the network interface batches up queries before sending them to the server. You can also provide the `batchMax` option to define the maximum amount of queries you want in one batch. When left empty this will send all queries within the `batchInterval` in one batch.
 
-<h3 id="BatchingExample">Query batching example</h3>
+<h3 id="MutationBatching">Mutation batching</h3>
+Apollo also lets you automatically run multiple mutations in one request, similar to queries, by batching multiple mutations into one request.
```
Comment (BenEmdon) [inline source/network.md]: Didn't know this. Very cool!

**Comment by ignacio-chiazzo** [comment]
Waiting for https://github.com/apollographql/apollo-client/pull/2324

**Comment by ignacio-chiazzo** [comment]
Closing in favour of https://github.com/apollographql/core-docs/pull/323

**Comment by ignacio-chiazzo** [comment]
https://github.com/apollographql/apollo-client/pull/2498

**Pull request closed without merging**